README.md

```diff
@@ -1,62 +1,181 @@
 # Exercism v3 Local Environment
 
-This provides a local environment for running Exercism on Docker. It is very much a WIP.
+Hello 👋
 
-## Getting Started
+This repository will provide you with a local setup of Exercise that you can use to develop any part of the project.
 
-Run the following line-by-line.
+It is built using [Docker](https://www.docker.com/), which you need to install, but it does not require any real Docker knowledge to use.
+
+Our aim is to get you to a working setup within 10-minutes from now (presuming you have a decent internet connection).
+It's also possible to get each piece of Exercism working on your local machine without this repository, but that will take lots more effort and work.
+
+## Basic setup
+
+The following instructions take you through getting the most basic setup working.
+We'll explain what actually happens below the hood, and how to configure things afterwards
+
+### Getting set up
+
+To start, run the following instructions line-by-line:
 
 ```bash
 # Make a directory to host all of Exercism within
 mkdir exercism
 
-# cd into it
+# Move into the new directory
 cd exercism
 
-# Clone this repo
+# Clone this repository onto your computer
 git clone git@github.com:exercism/v3-docker-compose.git
 
-# Clone any essential repos you plan to hack on
-git clone git@github.com:exercism/v3-website.git
-git clone git@github.com:exercism/tooling-orchestrator.git
-git clone git@github.com:exercism/tooling-invoker.git
-git clone git@github.com:exercism/csharp-test-runner.git
-
+# Move into the new directory
 cd v3-docker-compose
 
-# start with the default stack
+# Create your "stack" - the collection of parts of Exercism you
+# want to run locally. For now just copy the default file.
 cp stack.default.yml stack.yml
-# edit stack.yml to enabled the services (runners, analyzers, you'll be using)
-./bin/build_stack
 
-# Build everything in parallel
-docker-compose build --parallel
+# Start everything
+./bin/start
+```
+
+### Updating everything
+
+The first time you use `./bin/start` we will download the latest versions of all the Exercism components for you to use.
+When you want to upgrade those components, run:
+
+```bash
+docker-compose pull
+```
+
+If you face any issues getting started, we recommend running this step in case anything is cached locally from some previous installation.
```

Comment:
```suggestion
If you face any issues getting started, we recommend running this step in case anything is cached locally from a previous installation.
```